<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewItemWidgetClass</class>
 <widget class="QWidget" name="NewItemWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewItemWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemStockInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemPriceInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Item</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemNameInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stock Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="wipeSaveFileButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Wipe save file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="useSaveFileButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Append/use existing save file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitItemCreatorButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>263</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
